test/kernels/algorithms/test_qkernel_trainer.py

```diff
@@ -0,0 +1,102 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" Test QuantumKernelTrainer """
+
+from test import QiskitMachineLearningTestCase
+
+import unittest
+
+import numpy as np
+
+from qiskit import Aer
+from qiskit.circuit.library import ZZFeatureMap
+from qiskit.utils import algorithm_globals
+from qiskit.algorithms.optimizers import COBYLA
+from qiskit_machine_learning.kernels import QuantumKernel
+from qiskit_machine_learning.kernels.algorithms import QuantumKernelTrainer
+from qiskit_machine_learning.utils.loss_functions import SVCLoss
+from qiskit_machine_learning.algorithms.classifiers import QSVC
+
+
+class TestQuantumKernelTrainer(QiskitMachineLearningTestCase):
+    """Test QuantumKernelTrainer Algorithm"""
+
+    def setUp(self):
+        super().setUp()
+        algorithm_globals.random_seed = 10598
+        self.optimizer = COBYLA(maxiter=25)
+        self.backend = Aer.get_backend("statevector_simulator")
```

Comment:
Just noticed, please replace `statevector_simulator` with `aer_simulator_statevector`, otherwise there's a deprecation warning.
```suggestion
        self.backend = Aer.get_backend("aer_simulator_statevector")
```